docusaurus/docs/dev-docs/plugins/graphql.md

```diff
@@ -903,3 +903,70 @@ Using API tokens in the the GraphQL playground requires adding the authorization
 
 Replace `<TOKEN>` with your API token generated in the Strapi Admin panel.
 :::
+
+## Security
+
+As GraphQL is a query language, it allows users to use a wider pannel of inputs than traditional REST APIs.
+
+Due to this feature, GraphQL APIs are inherently prone to various security risks, but they can be reduced by taking appropriate precautions.
+Neglecting them can expose the API to vulnerabilities like credential leakage or denial of service attacks.
+
+### Disable introspection and playground in production
+
+In production, it is recommended to disable the GraphQL Playground and the introspection query.
+If you havent edited the [configuration file](/dev-docs/configurations/plugins#graphql-configuration), it must be already disabled.
+
+### Limit max depth and complexity
+
+The depth limit is the maximum number of nested fields that can be queried in a single request.
+During your test and depending on your use case, you can set the `depthLimit` option to a higher value.
+
+But a malicious user could send a query with a very high depth, which could your server to stop responding or crash.
+
+Review the [configuration file](/dev-docs/configurations/plugins#graphql-configuration) to set this options.
+
+### Install graphql armor
+
+[Graphql armor](https://github.com/Escape-Technologies/graphql-armor) is a GraphQL middleware that protects your GraphQL API from malicious queries.
+Installing it is simple and it prevent the most common attacks.
```

Comment:
```suggestion
Installing it is simple and prevents the most common attacks.
```

Reply:
Plus, per our [12 rules of technical writing - rule #6](https://handbook.strapi.io/user-success-manual/12-rules-of-technical-writing), we should never assume something is easy or difficult. It all depends on the user's experience.